Rollbar/RollbarConfiguration.h

```diff
@@ -35,7 +38,15 @@
 @property (readonly, atomic, copy) NSString *personId;
 @property (readonly, atomic, copy) NSString *personUsername;
 @property (readonly, atomic, copy) NSString *personEmail;
-@property (atomic, copy) void (^payloadModification)(NSDictionary *payload);
+
+// Modify payload
+@property (atomic, copy) void (^payloadModification)(NSMutableDictionary *payload);
+
+// Decides whether or not to send payload. Returns true to send, false
```

Comment:
Should be `Returns true to not send` or something like that as `Returns true to send` is incorrect. The code does the right thing, just this comment is off.

Reply:
Comment changed. The code should be correct.